bin-src/io/GraphQLClient.js

```diff
@@ -12,35 +15,46 @@ export default class GraphQLClient {
     this.headers.Authorization = `Bearer ${token}`;
   }
 
-  async runQuery(query, variables, headers) {
-    const response = await this.client.fetch(this.uri, {
-      body: JSON.stringify({ query, variables }),
-      headers: { ...this.headers, ...headers },
-      method: 'post',
-    });
-
-    const { data, errors } = await response.json();
-
-    if (errors) {
-      if (Array.isArray(errors)) {
-        errors.forEach((err) => {
-          // eslint-disable-next-line no-param-reassign
-          err.name = err.name || 'GraphQLError';
-          // eslint-disable-next-line no-param-reassign
-          err.at = `${err.path.join('.')} ${err.locations
-            .map((l) => `${l.line}:${l.column}`)
-            .join(', ')}`;
+  async runQuery(query, variables, { headers = {}, retries = 0 } = {}) {
```

Comment:
Adding another field for `retries` felt wrong so I threw it all in one object since `headers` is only used in one other place!